### PR TITLE
Vue3 Upgrade Redo Base Epic PR

### PR DESCRIPTION
**Pull request by M-Kusumgar**
I realised it was quite difficult to keep track of changes with my initial code and there were a lot of random changes to the config files (basically due to me copying and pasting from a fresh Vue 3 install and trying to get things to work at the start of this project). This will now be a base PR and I will create other PRs to merge into this one. This particular one just has package changes currently!

Merged PRs:
- None for now

Active PRs:
- #823
- #825 
- #829 

Upcoming PRs (not an exhaustive list and not in order):
- defineComponentVue2 in all vue components
- updating reside's Vue charts package
- updating reside's Vue dynamic form package
- recreating required bootstrap vue components using vanilla bootstrap

Proof of Concept:
#831

### CODE REVIEW

File: src/app/static/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "sass": "gulp sass && gulp inject",
     "js": "webpack",
-    "build": "gulp copy-templates && gulp styles && webpack",
+    "build": "vue-cli-service serve ./src/index.ts",
```
Comment (hillalex) [inline src/app/static/package.json:12]: `serve` implies that this starts the app, but this task just needs to build a bundle (that the kotlin backend will serve). Could add a separate `serve` task if useful for development.

Comment (M-Kusumgar) [reply src/app/static/package.json:12]: oh yh this was purely for my ease, i can add another command

**Review by EmmaLRussell — COMMENTED** [review]
Don't really have any comments beyond my previous review for the the other PR. 
The package.json still contains references to Vue2 packages like `bootstrap-vue`, which I assume will be removed in a future PR.

File: src/app/static/package.json (hunk 1 above)
Comment (M-Kusumgar) [reply src/app/static/package.json]: done!

**Pull request closed without merging**